about.html

```diff
@@ -0,0 +1,96 @@
+                    <li><i><img class="social" src="./assets/images/icons/twitter.svg" alt="twitter"></i></li>
+                    <li>English</li>
+                    <li>My Page</li>
+                    <li>Logout</li>
+                </ul>
+            </div>
+            <div class="secondRow">
+                <div class="logo"><a href="./index.html"> Global University</a></div>
+
+                <ul class="navbarLinks">
+                    <li class=""><a href="./about.html">About</a></li>
+                    <li class=""><a href="#">Program</a></li>
+                    <li class=""><a href="#">Join</a></li>
+                    <li class=""><a href="#">Sponsor</a></li>
+                    <li class=""><a href="#">News</a></li>
+                    <li class=""><a href="#">cc campaign</a></li>
+                </ul>
+            </div>
+
+        </nav>
+    </header>
+    <main id="main">
+        <section class="intro">
+            <div class="blur aboutBlur">
+                <img src="./assets/images/hamburger.svg" alt="hamburger" class="hamburger" id="humbarger">
+                <span class="welcome"> Global University</span>
+                <h1 class="title">Global University is the most famous University in the Europa</h1>
+                <div class="introDescription aboutIntroDescription">
+                    This year Global University accepts more than 3 thousand of students from out side of the Europa
+                    Our University stublished at 2000 and graduedtd around 1700 students from deffrent faculties.
+                </div>
+                <div class="introFooter introFooterAbout">Please contact us by email for further questions about global University at globaluniversity@gb.org</div>
+            </div>
+        </section>
+        <section class="aboutLogo">
+            <div class="aboutLogoTitle">Global University global summit 2015 logo</div>
+            <div class="line"></div>
+            <div class="aboutLogoDescription">
+                The logo of the global University was designed through the online competition and chosen in online selection
+            </div>
+         <div class="aboutLogoImg">  <img src="./assets/images/logo.png" alt="logo"></div>
+
+        </section>
+        <hr>
+        <section class="previousActivities">
+            <div class="previousActivitiesTitle">See the past graduated students</div>
+            <div class="line"></div>
+            <div class="previosActivitesContainer">
+                <div class="graduate graduate1">
+                    <div>2000</div>
+                    <div>BBA students at Global University</div>
+                </div>
+                <div class="graduate graduate2">
+                    <div>2005</div>
+                    <div>BCA students at Global University</div>
+                </div>
+            </div>
+
+        </section>
```

Comment:
- [ ] [optional] It would be nice if you ensured this section's color effect matches the one in the design:




Behance Design          |  Your design     
:-------------------------:|:-------------------------:
![image](https://user-images.githubusercontent.com/61408860/219745270-19cb9246-2b9c-4078-b314-917c71b4b828.png) | ![image](https://user-images.githubusercontent.com/61408860/219744928-1af3d96b-c319-4514-a350-4bae125b35a3.png)